microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -781,3 +781,44 @@ def verify_boot_error_fail_warnings(self, node: Node) -> None:
             "unexpected error/failure/warnings shown up in bootup log of distro"
             f" {node.os.name} {node.os.information.version}"
         ).is_empty()
+
+    @TestCaseMetadata(
+        description="""
+        This test will check ClientAliveInterval value in sshd config.
+
+        Steps:
+        1. Find ClientAliveInterval from sshd config.
+        2. Pass with warning if not find it.
+        3. Pass with warning if the value is not between 0 and 180.
+        """,
+        priority=1,
+    )
+    def verify_client_active_interval(self, node: Node) -> None:
+        ssh = node.tools[Ssh]
+        value = ssh.get("ClientAliveInterval")
+        if not value:
+            raise PassedException("not find ClientAliveInterval in sshd_config")
+        if not (int(value) > 0 and int(value) < 181):
+            raise PassedException("ClientAliveInterval should be set between 0 and 180")
+
+    @TestCaseMetadata(
+        description="""
+        This test will check no pre added users existing in vm.
+
+        Steps:
+        1. Exclude current user from all users' list.
+        2. Fail the case if the password of any above user existing.
+        """,
+        priority=1,
+    )
+    def verify_no_pre_exist_users(self, node: Node) -> None:
+        current_user = node.tools[Whoami].get_username()
+        cat = node.tools[Cat]
+        passwd_outputs = cat.read_with_filter(
+            "/etc/shadow", current_user, True, True, False
+        )
+        for passwd_raw_output in passwd_outputs.splitlines():
+            user_passwd = passwd_raw_output.split(":")[1]
+            user_name = passwd_raw_output.split(":")[0]
+            if not ("*" in user_passwd or "!" in user_passwd):
+                raise LisaException(f"password of user {user_name} is not deleted.")
```

Comment:
Describe it from positive way to say how to fix it, like 

password of user {user_name} should be deleted in /etc/shadow.